### ISSUE TITLE
ImportError: Fleet python client is not installed.

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#1081]
While performing a clean install of OVS & ASD MANAGERS:

```
root@ovs-node01-1604:~# asd-manager setup
+++++++++++++++++++++++++++
+++  ASD Manager setup  +++
+++++++++++++++++++++++++++
- Verifying distribution
2016-10-25 10:39:23 60600 +0200 - ovs-node01-1604 - 23139/140711298955008 - asd-manager/service - 0 - ERROR - No fleet client detected Fleet python client is not installed. Please check documentation on how to install it. No module named fleet.v1
Traceback (most recent call last):
  File "source/tools/services/service.py", line 77, in has_fleet_client
    from source.tools.services.fleetctl import FleetCtl
  File "source/tools/services/fleetctl.py", line 29, in <module>
    raise ImportError('Fleet python client is not installed. Please check documentation on how to install it. {0}'.format(ie))
ImportError: Fleet python client is not installed. Please check documentation on how to install it. No module named fleet.v1
```

**Comment by wimpers** [comment]
Anything special you did? Is this coming out of the nightly builds?

**Comment by khenderick** [comment]
It's just logging nothing that is "wrong".

Let's remove this fleet stuff; nobody is using it anyway, and if at one time in the far far future somebody actually wants to use it, we can re-implement it properly.

**Comment by wimpers** [comment]
If not giving issues, let's remove in Gilbert

**Comment by khenderick** [comment]
Fixed by #1115 and openvstorage/alba-asdmanager#145, packaged in openvstorage-2.7.4-rev.4256.dbf93f5 and openvstorage-sdm-1.6.4-rev.417.4458c4c

**Comment by JeffreyDevloo** [comment]
### Information

Did not encounter this error anymore during our automated setups.

### Steps
- Installation of a fresh cluster

### Output
- Ovs installed succesfully
- Asd-manager installed succesfully

No longer got the logging which could be ignored.

### Test result

Test passed.
